src/OpenTelemetry.Instrumentation.Owin/Implementation/DiagnosticsMiddleware.cs

```diff
@@ -199,11 +210,34 @@ private static void RequestEnd(IOwinContext owinContext, Exception exception)
 
             activity.Stop();
 
+            if (OwinInstrumentationMetrics.HttpClientDuration.Enabled)
+            {
+                var tags = new TagList
+                {
+                    { SemanticConventions.AttributeHttpMethod, owinContext.Request.Method },
+                    { SemanticConventions.AttributeHttpScheme, owinContext.Request.Scheme },
+                    { SemanticConventions.AttributeHttpStatusCode, owinContext.Response.StatusCode },
+                };
+
+                OwinInstrumentationMetrics.HttpClientDuration.Record(activity.Duration.TotalMilliseconds, tags);
+            }
+
             if (!(Propagators.DefaultTextMapPropagator is TraceContextPropagator))
             {
                 Baggage.Current = default;
             }
         }
+        else
+        {
+            if (stopwatch != null && OwinInstrumentationMetrics.HttpClientDuration.Enabled)
```

Comment:
nit: It looks like stopwatch will always be null when `!OwinInstrumentationMetrics.HttpClientDuration.Enabled` so you could probably skip that check here to pick up a little bit more perf.